lib/uo_cli.rb

```diff
@@ -188,66 +197,84 @@ def self.create_project_folder(dir_name)
+                    "https://raw.githubusercontent.com/urbanopt/urbanopt-cli/master/example_files/osm_building/7.osm",
+                    "https://raw.githubusercontent.com/urbanopt/urbanopt-cli/master/example_files/osm_building/8.osm",
+                    "https://raw.githubusercontent.com/urbanopt/urbanopt-cli/master/example_files/osm_building/9.osm"
+                ]
+
+                # Download weather file to user's local machine
+                remote_weather_files.each do |weather_file|
+                    weather_path, weather_name = File.split(weather_file)
+                    weather_download = open(weather_file, {ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE})
+                    IO.copy_stream(weather_download, File.join(weather_dir_abs_path, weather_name))
+                end
+
+                osm_files.each do |osm_file|
+                    osm_path, osm_name = File.split(osm_file)
+                    osm_download = open(osm_file, {ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE})
+                    IO.copy_stream(osm_download, File.join(osm_dir_abs_path, osm_name))
+                end
+
+                feature_path, feature_name = File.split(example_feature_file)
+                example_feature_download = open(example_feature_file, {ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE})
+                IO.copy_stream(example_feature_download, File.join(dir_name, feature_name))
 
-            config_path, config_name = File.split(config_file)
-            config_download = open(config_file, {ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE})
-            IO.copy_stream(config_download, File.join(dir_name, config_name))
+            end
         end
     end
 
 
     # Perform CLI actions
-    if @user_input[:project_folder]
-        create_project_folder(@user_input[:project_folder])
+    if @user_input[:project_folder] && @user_input[:empty_project_folder].nil?
+        create_project_folder(@user_input[:project_folder], empty_folder = false)
         puts "\nAn example FeatureFile is included: 'example_project.json'. You may place your own FeatureFile alongside the example."
         puts "Weather data is provided for the example FeatureFile. Additional weather data files may be downloaded from energyplus.net/weather for free"
         puts "If you use additional weather files, ensure they are added to the 'weather' directory. You will need to configure your mapper file and your osw file to use the desired weather file"
         puts "Next, move inside your new folder ('cd <FolderYouJustCreated>') and create ScenarioFiles using this CLI: 'uo -m -f <FFP>'"
+    elsif @user_input[:project_folder] && @user_input[:empty_project_folder]
+        create_project_folder(@user_input[:project_folder], empty_folder = true)
+        puts "Add your FeatureFile in the Project directory you just created."
+        puts "Add your weather data files in the Weather folder. They may be downloaded from energyplus.net/weather for free"
```

Comment:
Should there be an additional check while running the project to see if the weather folder is empty?

Reply:
I think that OpenStudio CLI will put something related to that in the run.log.  Also, there are other ways of specifying weather file locations (David can elaborate).   Maybe just a warning at the URBANopt CLI level would be nice, but not an error.